src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/xunit.runner.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://xunit.github.io/schema/current/xunit.runner.schema.json",
+    "longRunningTestSeconds": 61,
+    "diagnosticMessages": true,
+    "parallelizeAssembly": true
```

Comment:
@mikeharder:  This enables test parallelization within the XUnit runner.   The test suite went from ~21 minutes to ~3 minutes on average and seems to be holding up well in repeated local testing with the .NET core runner.

Reply:
Note: with the change to run diagnostics tests in isolation and serially, the suite run time is ~4 minutes now.

Reply:
This is great, I think <= 5 minutes is a good goal for a live test suite.